### DISCUSSION TITLE
这些快捷键 没用啊

### DISCUSSION THREAD

**Discussion by zhuhaisto** [tamlok/vnote#2170] [Q&A]
![QQ截图20220627071238](https://user-images.githubusercontent.com/108032980/175837501-3f5b7afa-99bc-42cf-bef0-4599fb00be12.jpg)
![QQ截图20220627071223](https://user-images.githubusercontent.com/108032980/175837508-3c307743-61b5-4853-b2f1-faa5f9415469.jpg)

** 反正就是没用啊 **

**Comment by tamlok** [comment]
你是怎么按的呢？

**Reply by zhuhaisto** [reply]
ctrl+g 不是说下个版本吗? 静等就可以了啊

**Comment by zhuhaisto** [comment]
啥时候快捷键能用啊 下个版本啥时候出啊 过了7天了

**Comment by tamlok** [comment]
ctrl-g, e 你是怎么按的？这个一直都支持

**Reply by zhuhaisto** [reply]
![图片](https://user-images.githubusercontent.com/108032980/177184417-1b7a92ba-883f-4137-be7c-b68e58050d91.png)
就是一个手指按住ctrl 一个手指再按g 没效果

**Reply by tamlok** [reply]
> ![图片](https://user-images.githubusercontent.com/108032980/177184417-1b7a92ba-883f-4137-be7c-b68e58050d91.png) 就是一个手指按住ctrl 一个手指再按g 没效果

右上角查看快捷键帮助，认真读一下？

**Reply by zhuhaisto** [reply]
我认为我快捷键使用没问题 不就是按住ctrl键 然后不松再按g吗？我也试过了 两个同时按 没先后顺序 反正就是没生效
我用ctrl + f 就行 ctrl+g 就不行 我操作应该没问题

**Reply by tamlok** [reply]
> 我认为我快捷键使用没问题 不就是按住ctrl键 然后不松再按g吗？我也试过了 两个同时按 没先后顺序 反正就是没生效 我用ctrl + f 就行 ctrl+g 就不行 我操作应该没问题

那什么时候按e？看一下快捷键说明呀
